Source/WebKit/UIProcess/Downloads/DownloadProxy.cpp

```diff
@@ -154,6 +154,8 @@ void DownloadProxy::didReceiveData(uint64_t bytesWritten, uint64_t totalBytesWri
 
 void DownloadProxy::decideDestinationWithSuggestedFilename(const WebCore::ResourceResponse& response, String&& suggestedFilename, CompletionHandler<void(String, SandboxExtension::Handle, AllowOverwrite)>&& completionHandler)
 {
+    RELEASE_LOG_INFO_IF(!response.expectedContentLength(), Network, "DownloadProxy::decideDestinationWithSuggestedFilename expectedContentLength is null");
```

Comment:
If this is unexpected, should this be RELEASE_LOG_ERROR_IF instead?

Reply:
It is not unexpected (if the response does not have content-length, we do not have an estimate) but it is a case that is interesting to flag.